Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -620,6 +620,19 @@ theorem rescale_mul (a b : R) : rescale (a * b) = (rescale b).comp (rescale a) :
   ext
   simp [← rescale_rescale]
 
+theorem rescale_map_eq_map_rescale_of_ringHom {S : Type*} [CommSemiring S] (φ : R →+* S) (r : R)
+    (f : R⟦X⟧) : rescale (φ r) (PowerSeries.map φ f) =
+      PowerSeries.map (φ : R →+* S) (rescale r f) := by
+  ext n
+  simp [coeff_rescale, coeff_map, map_mul, map_pow]
+
+theorem rescale_map_eq_map_rescale {A S : Type*} [CommSemiring A] [Algebra A R] [CommSemiring S]
+    [Algebra A S] (φ : R →ₐ[A] S) (a : A) (f : R⟦X⟧) :
+    rescale (algebraMap A S a) (PowerSeries.map φ f) =
+      PowerSeries.map (φ : R →+* S) (rescale (algebraMap A R a) f) := by
```

Comment:
```suggestion
    rescale (algebraMap A S a) (f.map φ) =
      (rescale (algebraMap A R a) f).map (φ : R →+* S) := by
```
And I am guessing the `: R →+* S` coercion is also removable but I've not tried.